src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```
@@ -60,6 +60,9 @@
     <AndroidManifest Condition=" '$(AndroidManifest)' == '' and Exists ('Properties\AndroidManifest.xml') and !Exists ('AndroidManifest.xml') ">Properties\AndroidManifest.xml</AndroidManifest>
     <AndroidManifest Condition=" '$(AndroidManifest)' == '' ">AndroidManifest.xml</AndroidManifest>
     <GenerateApplicationManifest Condition=" '$(GenerateApplicationManifest)' == '' ">true</GenerateApplicationManifest>
+
+    <!-- Enable/Disable default ruuntime features-->
+    <UseNativeHttpHandler>true</UseNativeHttpHandler>
```

I think this should be:

```suggestion
    <UseNativeHttpHandler Condition=" '$(UseNativeHttpHandler)' == '' ">true</UseNativeHttpHandler>
```

So that users can override it in their `.csproj` file.